client/.prettierignore

```diff
@@ -0,0 +1,6 @@
+# Ignore images
+*.svg
+
+# Ignore vender css
+client/src/css/justgrid.css
```

Comment:
Would it make sense to create a `client/src/css/vendor` directory to isolate vendor CSS from local CSS in a more explicit way (and make it easier to drop in more if we need to)?

Also, if I'm not mistaken, justgrid.css was actually developed by @crhallberg (see [https://crhallberg.com/justgrid/](https://crhallberg.com/justgrid/)). I wonder if we should add a header comment to the file to indicate its source/license.

Reply:
Sorry, this was copy/paste.

Reply:
Made change